>> anaconda-3-5.3.1 has been loaded.
>> python-3.8.2 has been loaded.
>> cuda-10.1_cudnn-7.6.5 has been loaded.
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/10151>
distutils: /home/arlan.uy/.local/include/python3.7m/UNKNOWN
sysconfig: /home/arlan.uy/.local/include/python3.7/UNKNOWN
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 95, in program_main
    ga_params = create_empty_yaml(runs, generations, population_size, savefile_name)
  File "main.py", line 24, in create_empty_yaml
    ga_params['run_total'] = runs
TypeError: 'NoneType' object does not support item assignment
srun: error: saliksik-gpu-06: task 6: Exited with exit code 1
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 303, in simulate
    self.evaluate(chrom, function)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 253, in evaluate
    chromosome['fitness'] += assign_fitness(chromosome['system'].out_spiketrain, pair['output'], function)/minlen*100
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 41, in assign_fitness
    result = lc_substring(string_format(generated), string_format(actual))
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/fitness.py", line 29, in lc_substring
    timer_write("Evaluate", start, finish)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 28, in timer_write
    timer_params = conf_load(filename)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 22, in conf_load
    return timer_params
UnboundLocalError: local variable 'timer_params' referenced before assignment
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 33, in timer_write_run
    timer_params = conf_load(filename)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 22, in conf_load
    return timer_params
UnboundLocalError: local variable 'timer_params' referenced before assignment
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 33, in timer_write_run
    timer_params = conf_load(filename)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 22, in conf_load
    return timer_params
UnboundLocalError: local variable 'timer_params' referenced before assignment
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 33, in timer_write_run
    timer_params = conf_load(filename)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 22, in conf_load
    return timer_params
UnboundLocalError: local variable 'timer_params' referenced before assignment
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 33, in timer_write_run
    timer_params = conf_load(filename)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 22, in conf_load
    return timer_params
UnboundLocalError: local variable 'timer_params' referenced before assignment
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 273, in simulate
    timer_write_run(run_index)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 33, in timer_write_run
    timer_params = conf_load(filename)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 22, in conf_load
    return timer_params
UnboundLocalError: local variable 'timer_params' referenced before assignment
srun: error: saliksik-gpu-06: tasks 0,7: Exited with exit code 1
srun: error: saliksik-gpu-06: tasks 2-5: Exited with exit code 1
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 188, in program_main
    gaframework(system, test_cases_path, savefile_name)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 26, in gaframework
    execute_experiment(rssnp, ga, gaeval, ga_params, loadfile_name, start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/experimenter_uncommented.py", line 49, in execute_experiment
    gaeval.run(ga, rssnp, stats_run['population_size'], stats_run['fitness_function'],  gaeval.no_of_run, gaeval.no_of_gen, stats_run['mutation_rate'], loadfile_name, stats_run['selection_func'], start_new, start_from_gen)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/GAevaluator_integ.py", line 38, in run
    run_fitness = genetic_algo.simulate(system, size, function, generations, mutation_rate, path_name, run, selection_func, start_from_gen)            
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 303, in simulate
    self.evaluate(chrom, function)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 253, in evaluate
    chromosome['fitness'] += assign_fitness(chromosome['system'].out_spiketrain, pair['output'], function)/minlen*100
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/geneticalgo_integ.py", line 41, in assign_fitness
    result = lc_substring(string_format(generated), string_format(actual))
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/fitness.py", line 29, in lc_substring
    timer_write("Evaluate", start, finish)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 28, in timer_write
    timer_params = conf_load(filename)
  File "/scratch1/scratch2/arlan.uy/ERSSNP_CPU_GPU_Framework/main_dir/src/abstracts/cpu_timer.py", line 22, in conf_load
    return timer_params
UnboundLocalError: local variable 'timer_params' referenced before assignment
srun: error: saliksik-gpu-06: task 1: Exited with exit code 1
